app/views/homepage/_location_bar.haml

```diff
@@ -8,12 +8,17 @@
     function initialize() {
       var input = document.getElementById('q');
       var autocomplete = new google.maps.places.Autocomplete(input);
+      autocomplete.setTypes(['geocode']);
 
       google.maps.event.addListener(autocomplete, 'place_changed', function(event){
         var place = autocomplete.getPlace();
-        if(place != null && place.name != "") {
-          console.log('name:', place.name, '\naddress: ', place.formatted_address, '\ncoordinates: ', place.geometry.location.toUrlValue());
-          document.getElementById('lc').value = place.geometry.location.toUrlValue();
+        if(place != null) {
+          if( place.geometry != null ) {
+            console.log('name:', place.name, '\naddress: ', place.formatted_address, '\ncoordinates: ', place.geometry.location.toUrlValue());
```

Comment:
Ok, I understand. But do we really want to have them also in production? To me it gives a bit unprofessional feeling if I see a site that is recording test data to the console on production.